Pyrado/tests/test_policies.py

```diff
@@ -656,6 +656,29 @@ def old_evaluate(rollout: StepSequence, hidden_states_name: str = "hidden_states
 )
 def test_twoheaded_policy_evaluate_packed_padded_sequences(env: Env, policy: RecurrentPolicy):
```

Comment:
nice test

Reply:
thanks, however this function already existed, I just put some life into it